packages/lending/src/protocols/spark/configs.ts

```diff
@@ -0,0 +1,191 @@
+import * as common from '@protocolink/common';
+import { gnosisTokens, mainnetTokens } from './tokens';
+
+export const ID = 'spark';
+export const DISPLAY_NAME = 'Spark';
+
+export interface Reserve {
+  asset: common.Token;
+  aToken: common.Token;
+  used: {
+    deposit?: boolean;
+    withdraw?: boolean;
+    borrow?: boolean;
+    repay?: boolean;
+    flashLoan?: boolean;
+  };
+}
+
+type ContractName = 'Pool' | 'PoolDataProvider' | 'AaveOracle';
+
+interface Config {
+  chainId: number;
+  contractMap: Record<ContractName, string>;
+  reserves: Reserve[];
+}
+
+export const configs: Config[] = [
+  // https://devs.spark.fi/deployment-addresses/ethereum-addresses
+  {
+    chainId: common.ChainId.mainnet,
+    contractMap: {
+      Pool: '0xC13e21B648A5Ee794902342038FF3aDAB66BE987',
+      PoolDataProvider: '0xFc21d6d146E6086B8359705C8b28512a983db0cb',
+      AaveOracle: '0x8105f69D9C41644c6A0803fDA7D03Aa70996cFD9',
+    },
+    reserves: [
+      {
+        asset: mainnetTokens.ETH,
+        aToken: mainnetTokens.spWETH,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true },
+      },
+      {
+        asset: mainnetTokens.DAI,
+        aToken: mainnetTokens.spDAI,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
+      },
+      {
+        asset: mainnetTokens.WETH,
+        aToken: mainnetTokens.spWETH,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
+      },
+      {
+        asset: mainnetTokens.wstETH,
+        aToken: mainnetTokens.spwstETH,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
+      },
+      {
+        asset: mainnetTokens.USDC,
+        aToken: mainnetTokens.spUSDC,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
+      },
+      {
+        asset: mainnetTokens.WBTC,
+        aToken: mainnetTokens.spWBTC,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
+      },
+    ],
+  },
+  // https://devs.spark.fi/deployment-addresses/gnosis-addresses
+  {
+    chainId: common.ChainId.gnosis,
+    contractMap: {
+      Pool: '0x2Dae5307c5E3FD1CF5A72Cb6F698f915860607e0',
+      PoolDataProvider: '0x2a002054A06546bB5a264D57A81347e23Af91D18',
+      AaveOracle: '0x8105f69D9C41644c6A0803fDA7D03Aa70996cFD9',
+    },
+    reserves: [
+      {
+        asset: gnosisTokens.xDAI,
+        aToken: gnosisTokens.spWXDAI,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
```

Comment:
native token can not flash loan
can ref [line 95](https://github.com/dinngo/protocolink-js-sdk/blob/master/packages/lending/src/protocols/aave-v3/configs.ts#L95)